manifests/1.1.0/manifest.linux.x64.1.1.0.yml

```diff
@@ -0,0 +1,36 @@
+---
+manifest:
+  schema-version: 1.0
+build:
+  version: 1.1.0
+  platform: linux
+  architecture: x64
+products:
+#  opensearch: 
```

Comment:
You should delete this and not comment out. We have git for history.

Reply:
Removed.